### PR TITLE
Removed unnecessary features

### PR DESCRIPTION
**Pull request by eavuong23**
Removed:
- rest api interface (for users creating their own name -> ip)
- storing dns query results to a database (removed the db entirely since it won't be used)
- all views

Changed:
- docker setup (host port 53 -> container port 10053)
- fixed ip directing to the proxy (192.168.0.1 for now)
- using newer version of django and twisted

### CODE REVIEW

**Comment by rays** [comment]
We can use our travis account, so put back in/or replace travis stuff. Have travis test the code (add tests if they were removed

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM python:2.7
+ENV PYTHONUNBUFFERED 1
+run mkdir /code
```
Comment (denniskpw) [inline Dockerfile]: `RUN`?

**Comment by eavuong23** [comment]
I removed all the tests that do not apply anymore. There's only 1 test that applies.

**Comment by denniskpw** [comment]
🚀

**Comment by denniskpw** [comment]
Looks good so far.

**Comment by rays** [comment]
💯

**Pull request merged**